docs/concepts/output-accessibility.md

```diff
@@ -0,0 +1,118 @@
+  friendly for screen readers, enhancing the accessibility of your scripts.
+ms.custom: experience
+ms.date: 03/25/2025
+title: Improve the accessibility of DSC output in PowerShell
+---
+
+# Improve the accessibility of DSC output in PowerShell
+
+Most terminal environments only display raw text. Users that rely on screen readers are faced with
+tedious narration when consuming large amounts of raw text because the raw output doesn't have the
+accessibility metadata to characterize the format of the content.
+
+There are two ways to improve the accessibility of the output in PowerShell:
+
+- Output the data in a way that it can be viewed in another tool that supports screen reading
+  technologies.
+- Reduce the amount of output displayed in the terminal by filtering and selecting the data you
+  want and output the text in a more readable format.
+
+## Display the data in a tool outside of the terminal
+
+For large amounts of data, rather than output to the host, consider writing output in a format that
+can be viewed in another tool that supports screen reading technologies. You might need to save the
+data to a file in a format that can be opened in another application.
+
+### Out-GridView command on Windows
+
+For small to moderate size output, use the `Out-GridView` command. The output is rendered using
+Windows Presentation Foundation (WPF) in tabular form, much like a spreadsheet. The GridView
+control allows you to sort, filter, and search the data, which reduces the amount of data that
+needs to be read. The GridView control is also accessible to screen readers. The **Narrator** tool
+built into Windows is able to read the GridView details, including column names and row count.
+
+The following example shows how to display a list of DSC resources in a GridView control.
+
+```powershell
+dsc resource list | ConvertFrom-Json | Out-GridView
+```
+
+The following example shows how to display a list of DSC adapted resources in a GridView control.
+
+```powershell
+dsc resource list -a Microsoft.Windows/WindowsPowerShell |
+    ConvertFrom-Json |
+    Out-GridView
+```
+
+The `Out-GridView` command is only available in PowerShell on Windows.
+
+### Character Separated Value (CSV) format
+
+Spreadsheet applications such as **Microsoft Excel** support CSV files. The following example shows
+how to save the output of a command to a CSV file.
+
+```powershell
+dsc resource list | ConvertFrom-Json | Export-Csv -Path .\myFile.csv
+Invoke-Item .\myFile.csv
+```
+
+The `Invoke-Item` command opens the file in the default application for CSV files.
+
+### HyperText Markup Language (HTML) format
+
+HTML files can be viewed by web browsers such as **Microsoft Edge**. The following example shows
+how to save the output of a command to an HTML file.
+
+```powershell
+dsc resource list | ConvertFrom-Json | ConvertTo-HTML | Out-File .\myFile.html
+Invoke-Item .\myFile.html
+```
+
+The `Invoke-Item` command opens the file in your default web browser.
+
```

Comment:
```suggestion
```

unfortunately we need to remove these until an issue is resolved in PWSH

Reply:
@theJasonHelmick - how does removing this affect the accessibility concerns?

Maybe we just note that this work in Windows PowerShell and link to the issue / remove the notice when that issue is resolved?

Reply:
The html written (5.1 or 7) as output does not include language tags so screen readers have to assume system language. We can’t use this, as a result.